design-library/src/components/BccItemTile/BccItemTile.css

```diff
@@ -0,0 +1,7 @@
+@layer components {
+  .bcc-item-tile {
+    height: 125px;
```

Comment:
Is this hardcoded height intended?

Reply:
How does the component behave with a lot of text? Maybe truncating needs to be done. A story for that could be nice to show it to users.

Reply:
Good point, right now horizontal overlap is truncated and vertical overlap is just hidden.